socorro/processor/rules/general.py

```diff
@@ -131,23 +131,6 @@ def action(self, raw_crash, dumps, processed_crash, processor_meta_data):
         processed_crash["os_version"] = os_ver
 
 
-class CollectorInfoRule(Rule):
-    """Copy keys from the raw crash related to the collector
-
-    Note: These are not validated or normalized. It's assumed the collector is
-    generating this data correctly.
-
-    """
-
-    KEYS = [
-        "collector_notes",
-    ]
-
-    def action(self, raw_crash, dumps, processed_crash, processor_meta_data):
-        for key in self.KEYS:
-            processed_crash[key] = raw_crash.get(key, None)
```

Comment:
This was never added to the pipeline, so I removed the code. However, we do need to add it.

Since the `collector_notes` value is an array of strings and I don't currently have support for that in `CopyFromRawCrashRule`, I couldn't just add this in this pass. I'll write up a separate bug for it.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1784558